tests/e2e/transaction/helper.ts

```diff
@@ -87,6 +95,63 @@ export async function fundAccounts(aptos: Aptos, accounts: Array<Account>) {
   return response;
 }
 
+export async function balance(aptos: Aptos, address: AccountAddress): Promise<number> {
+  type Coin = { coin: { value: string } };
+  const resource = await aptos.getAccountResource<Coin>({
+    accountAddress: address,
+    resourceType: "0x1::coin::CoinStore<0x1::aptos_coin::AptosCoin>",
+  });
+  return Number(resource.coin.value);
+}
+
+const fetchToken = (index: number, rsaPrivateKey: string, ephemKeyPair: EphemeralKeyPair) => {
+  const payload = {
+    iss: "test.oidc.provider",
+    aud: "test-keyless-dapp",
+    sub: `test-user-${index}`,
+    email: "test@aptoslabs.com",
+    email_verified: true,
+    iat: Math.floor(new Date().getTime() / 1000),
+    exp: 2700000000,
+    nonce: ephemKeyPair.nonce,
+  };
+  return sign(payload, rsaPrivateKey, { algorithm: "RS256", keyid: "test-rsa" });
+};
+
+export async function fetchTestKeylessAccount(
+  aptos: Aptos,
+  index: number,
+  rsaPrivateKey: string,
+): Promise<KeylessAccount> {
+  const ephemeralKeyPair = EphemeralKeyPair.generate();
+  const jwt = fetchToken(index, rsaPrivateKey, ephemeralKeyPair);
+  const account = await aptos.deriveKeylessAccount({ jwt, ephemeralKeyPair });
+  return account;
+}
+
+export function fetchDevnetTestKeylessAccount(index: number): KeylessAccount {
```

Comment:
why is it called `fetchDevnet...`? dont see anything devnet here...

Reply:
The proofs are specific to devnet.  Devnet has a different VK and or JWK set.